internal/stack/serverless.go

```diff
@@ -286,25 +297,25 @@ func (sp *serverlessProvider) composeProjectName() string {
 	return DockerComposeProjectName(sp.profile)
 }
 
-func (sp *serverlessProvider) localAgentComposeProject() (*compose.Project, error) {
+func (sp *serverlessProvider) localServiceComposeProject() (*compose.Project, error) {
 	composeFile := sp.profile.Path(profileStackPath, SnapshotFile)
 	return compose.NewProject(sp.composeProjectName(), composeFile)
 }
 
-func (sp *serverlessProvider) startLocalAgent(options Options, config Config) error {
+func (sp *serverlessProvider) startLocalServices(options Options, config Config) error {
 	err := applyServerlessResources(sp.profile, options.StackVersion, config)
 	if err != nil {
-		return fmt.Errorf("could not initialize compose files for local agent: %w", err)
+		return fmt.Errorf("could not initialize compose files for local service: %w", err)
 	}
 
-	project, err := sp.localAgentComposeProject()
+	project, err := sp.localServiceComposeProject()
```

Comment:
Nit.
As there could be more than one, would it be ok to rename it ? 
```suggestion
	project, err := sp.localServicesComposeProject()
```